package.json

```diff
@@ -3,20 +3,23 @@
   "email": "trust2065@gmail.com",
   "license": "UNLICENSED",
   "scripts": {
-    "start": "set NODE_ENV=production&&webpack --mode=development",
-    "dev": "set NODE_ENV=development&&webpack --mode=development --hot",
-    "build": "set NODE_ENV=production&&webpack --mode=production"
+    "start": "webpack-dev-server --open",
+    "build": "webpack --mode=production"
   },
   "devDependencies": {
+    "css-loader": "^2.1.1",
+    "file-loader": "^4.0.0",
+    "fullpage.js": "^3.0.5",
```

Comment:
Though the definition of `dependencies` is for the packages which execute in runtime. For frontend code, there is no package runs in runtime. However, it's a fashion that more developers separate non-build related packages like `fullpage.js` to `dependencies`

It's nothing correct or wrong. As long as you can explain why.

Reply:
I read some articles for `dependencies` versus `devDependencies`.
And yes, packages such as `fullpage.js` and `lodash` should belong to `dependencies`.